docs/release_notes/9_5.md

```diff
@@ -146,7 +124,7 @@ Many known issues exist in 9.5 including those related to:
 * Supportability
 * Containers
 
-Review the [upstream listing](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/known-issues) for the complete details on these issues.
+Review the [upstream listing](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#__WKANCHOR_5g) for the complete details on these issues.
```

Comment:
The link to RedHat Known Issues is wrong.  It should be WKANCHOR_5i  (not 5g)

Reply:
God you covered 👍🏻